src/main/java/com/contentful/java/cma/interceptor/ContentTypeInterceptor.java

```diff
@@ -38,18 +37,20 @@ public ContentTypeInterceptor(String contentType) {
   @Override public Response intercept(Chain chain) throws IOException {
     final Request request = chain.request();
 
-    final Request.Builder builder = request.newBuilder()
-        .addHeader(HEADER_NAME, contentType);
+    String requestContentType = isPatch(request) ? PATCH_CONTENT_TYPE : this.contentType;
```

Comment:
Don't like such hidden logic here, may be it is matter of taste.
I would implement it something like this:
```
addInterceptor(new ContentTypeInterceptor(DEFAULT_CONTENT_TYPE, request -> !"PATCH".equals(request.method())))
addInterceptor(new ContentTypeInterceptor(PATCH_CONTENT_TYPE, request -> "PATCH".equals(request.method())))
...
public ContentTypeInterceptor(String contentType, Predicate<Request> predicate) {...}
```